package.json

```diff
@@ -50,6 +51,7 @@
     "cross-env": "^5.0.5",
     "csurf": "^1.9.0",
     "express": "^4.15.3",
+    "formidable": "^1.1.1",
```

Comment:
are you using this?

Reply:
I am going back and forth between two of them trying to get the behaviour I want. which ever one finally works will mean I remove the other